pkg/util/kubernetes/hpa/datadog_external.go

```diff
@@ -0,0 +1,50 @@
+package hpa
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"gopkg.in/zorkian/go-datadog-api.v2"
+
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// QueryDatadogExtra converts the metric name and labels from the HPA format into a Datadog metric.
+// It returns the last value for a bucket of 5 minutes,
+func QueryDatadogExtra(metricName string, tags map[string]string) (int64, error) {
```

Comment:
What does the `Extra` indicate in the function name? :P

Reply:
lol, yeah I'm updating